### ISSUE TITLE
XenServer: Temporary refactor to use [xe.exe] for Windows

### ISSUE THREAD

**Issue by jefflill** [nforgeio/neonKUBE#1132]
This is a temporary workaround for #1130 which will work on all platforms.

For now, we really need to concentrate on getting a Windows neonDESKTOP demo working, and #1130 will probably take 2-3 days to do this right. where as porting to use the the **xe.exe** client we discovered for Windows should take only an hour or two of work.

I'm going to go with the **xe.exe** solution for now (packaging the binaries as content in the nuget package) and put #1130 on the backlog until we start work on neonDESKTOP for OS/X.

**Comment by jefflill** [comment]
I started the implementation and ran into a bit of a blocker I forgot about when I implemented this initially.

The problem is that although **xe-cli** has commands to create/remove ISO storage repositories, it doesn't have commands to add or remove files within these repositories.  The server expects those file to be created by Linux level file copying or via SCP.  This means we'll need to SSH/SCP into the machine to do this, which I was hoping to avoid (so we don't have to mess with SUDOERS).

I'm going to resolve this by using stock SSH.NET to manage the ISO file via SCP rather than using `LinuxSshProxy`, so we won't mess with the SUDOERS.  This isn't too bad.
